application/src/test/java/com/knubisoft/application/ApplicationTests.java

```diff
@@ -1,13 +1,59 @@
 package com.knubisoft.application;
 
+import com.knubisoft.application.ecryption.Encryption;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.security.PrivateKey;
+
 @SpringBootTest
 class ApplicationTests {
 
+    @Autowired
+    private Encryption encryption;
+    @Value("${keystore.password}")
+    private char[] keystorePassword;
```

Comment:
Additional to the first comment: you can save your credentials here like you did with original_data.